src/errors.ts

```diff
@@ -55,19 +55,28 @@ export interface ForbiddenErrorAPIResponse {
 }
 
 /**
- * The response returned by the Prismic Custom Types API bulk transaction
- * endpoint when a soft or hard limit is reached.
+ * The response returned by the Prismic Custom Types API bulk update transaction
+ * endpoint when changes contain a deletion operation of custom type(s) with
+ * existing document(s).
  */
-type BulkTransactionErrorAPIResponse = {
-	details: {
-		customTypes: {
-			id: string;
-			numberOfDocuments: number;
-			url: string;
-		}[];
-	};
+type BulkUpdateTransactionErrorAPIResponse = {
```

Comment:
💡 #idea: Aligning the name with the error would help with readability:

```diff
- type BulkUpdateTransactionErrorAPIResponse = {
+ type BulkUpdateHasExistingDocumentsErrorAPIResponse = {
```

(`BulkTransactionErrorAPIResponse` was improperly named.)

Reply:
Done